ENVIRONMENT.md

```diff
@@ -0,0 +1,24 @@
+| `GRAPH_ENVIRONMENT_DEV_CONFIG`            | [NOTE]: Only used for Dev environments. Add the graph environment config in JSON format only used for Dev environments. Be careful to escape any inner quotes as this is in a .env file and will be parsed by the shell. | JSON [(example)](./env.template) | Y (if 'Dev') |         |
+| `GRAPH_ENVIRONMENT_TYPE`                  | Graph environment type. This can be 'Dev' or 'Rococo' or 'Mainnet'.                                                                                                                                                      |       Dev\|Rococo\|Mainnet       |      Y       |         |
+| `HEALTH_CHECK_MAX_RETRIES`                | Number of `/health` endpoint failures allowed before marking the provider webhook service down                                                                                                                           |               >= 0               |              |   20    |
+| `HEALTH_CHECK_MAX_RETRY_INTERVAL_SECONDS` | Number of seconds to retry provider webhook `/health` endpoint when failing                                                                                                                                              |               > 0                |              |   64    |
+| `HEALTH_CHECK_SUCCESS_THRESHOLD`          | Minimum number of consecutive successful calls to the provider webhook `/health` endpoint before it is marked up again                                                                                                   |               > 0                |              |   10    |
+| `PROVIDER_ACCESS_TOKEN`                   | An optional bearer token authentication to the provider webhook                                                                                                                                                          |              string              |              |         |
+| `PROVIDER_ACCOUNT_SEED_PHRASE`            | Seed phrase for provider MSA control key                                                                                                                                                                                 |              string              |      Y       |         |
+| `PROVIDER_BASE_URL`                       | Base URL for provider webhook endpoints                                                                                                                                                                                  |               URL                |      Y       |         |
+| `PROVIDER_ID`                             | Provider MSA ID                                                                                                                                                                                                          |             integer              |      Y       |         |
```

Comment:
Do we do a initialization check to make sure that our Provider Id is correct? If not, we should add that as a todo for this and other gateway services with a key. It is a good double check that we are connecting to the correct network.

Reply:
No, we don't... I'll add a ticket